src/args.rs

```diff
@@ -9,13 +9,47 @@ use serde::Deserialize;
 
 use crate::git;
 
+/// Styles from https://github.com/rust-lang/cargo/blob/master/src/cargo/util/style.rs
+mod style {
+    use anstyle::*;
+    use clap::builder::Styles;
+
+    const HEADER: Style = AnsiColor::Green.on_default().effects(Effects::BOLD);
+    const USAGE: Style = AnsiColor::Green.on_default().effects(Effects::BOLD);
+    const LITERAL: Style = AnsiColor::Cyan.on_default().effects(Effects::BOLD);
+    const PLACEHOLDER: Style = AnsiColor::Cyan.on_default();
+    const ERROR: Style = AnsiColor::Red.on_default().effects(Effects::BOLD);
+    const VALID: Style = AnsiColor::Cyan.on_default().effects(Effects::BOLD);
+    const INVALID: Style = AnsiColor::Yellow.on_default().effects(Effects::BOLD);
+
+    pub const STYLES: Styles = {
+        Styles::styled()
+            .header(HEADER)
+            .usage(USAGE)
+            .literal(LITERAL)
+            .placeholder(PLACEHOLDER)
+            .error(ERROR)
+            .valid(VALID)
+            .invalid(INVALID)
+            .error(ERROR)
+    };
+}
+
+mod heading {
+    pub const GIT_PARAMETERS: &str = "Git Parameters";
+    pub const TEMPLATE_SELECTION: &str = "Template Selection";
+    pub const OUTPUT_PARAMETERS: &str = "Output Parameters";
```

Comment:
I wasn't 100% sure of good names for these headings, and there's a few options that I wasn't entirely sure whether to leave them in the main options section or in the output params (e.g. overwrite, allow-commands)

Reply:
for me it sounds reasonable.